style.css

```diff
@@ -333,3 +335,98 @@ body {
 hr {
   margin: 0;
 }
+
+.contact-me {
+  border-radius: 25px 0 0 0;
+  border: 1px;
+  background-color: #6070ff;
+  top: 70px;
+  color: #fff;
+  text-align: center;
+  padding: 70px 16px 20px 16px;
+}
```

Comment:
- [ ] I congratulate you upon achieving the `border-radius`. But still, we need to stick to the Figma design as much as possible. And for this reason, I suggest that you increase the border radius to match the Figma design.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/220331405-af289437-dd08-48f0-849a-a7983bbfdd39.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/220331564-dcd18d0c-839c-4936-9a36-265112381566.png)